test/image/compare_images

```diff
@@ -21,6 +21,9 @@ do
   format="${filename##*.}"
 
   case "$format" in
+    json)
+      continue # not exactly an image, so let's skip it
+      ;;
```

Comment:
As soon as we support deterministic identifiers in plotly.js, the JSON produced in `test-image` should be compared to baselines. In the meantime, we simply don't compare JSON to anything. However, they can be manually inspected if necessary since they are build artifacts of the `test-image` job.

Reply:
JSON are now diffed against baselines. I will resolve this one.